4-WebApp-your-API/Client/Startup.cs

```diff
@@ -35,6 +36,12 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddOptions();
 
+            // This is required to be instantiated before the OpenIdConnectOptions starts getting configured.
+            // By default, the claims mapping will map claim names in the old format to accommodate older SAML applications.
+            // 'http://schemas.microsoft.com/ws/2008/06/identity/claims/role' instead of 'roles'
+            // This flag ensures that the ClaimsIdentity claims collection will be built from the claims in the token
+            JwtSecurityTokenHandler.DefaultMapInboundClaims = false;
```

Comment:
>JwtSecurityTokenHandler.DefaultMapInboundClaims = false; [](start = 12, length = 56)

Adding this as it keeps the sample "modern". Presence of these older SAML age claims only confuse develoeprs

Reply:
Leaving it commented for now. We are not 100% sure

Reply:
leaving it commented for now

---
In reply to: [309059613](https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2/pull/147#discussion_r309059613) [](ancestors = 309059613)